app/src/main/res/layout/item_attachment.xml

```diff
@@ -90,9 +50,7 @@
             app:layout_constraintBottom_toBottomOf="parent"
             app:layout_constraintDimensionRatio="1"
             app:layout_constraintEnd_toEndOf="parent"
-            app:layout_constraintStart_toEndOf="@id/textsLayout"
-            app:layout_constraintTop_toTopOf="parent"
-            tools:visibility="visible" />
```

Comment:
I don't think you want to remove this constraint "top to top", no? 

Also, can we keep the tools:visibility if you have no preferences on the subject? I like it better if we can see it here